src/plugins/vis_type_table/jest.config.js

```diff
@@ -11,4 +11,5 @@ module.exports = {
   rootDir: '../../..',
   roots: ['<rootDir>/src/plugins/vis_type_table'],
   testRunner: 'jasmine2',
+  collectCoverageFrom: ['<rootDir>/src/plugins/vis_type_table/**/*.{js,ts,tsx}'],
```

Comment:
Are you sure we want to commit that?  My objection is that if all plugins enable coverage generation, this will increase the already long CI execution time. If you feel that we really need it, probably it make sense to do it in a separate PR

also if I'm not wrong with `collectCoverateFrom` we should also user `collectCoverage`: `true/false` param

Reply:
This string doesn't affect the CI somehow, since the CI doesn't use `--coverage` flag.
This setting is only handy for local testing. So you can open a terminal right in the plugin folder and run `yarn test:jest --coverage`, test coverage will be collected just for the `vis_type_table` plugin.
If you skip the setting, the global one will be used (`packages/kbn-test/jest-preset.js`), which will collect all of the files, including the plugin's one.